java-support/src/main/scala/io/cloudstate/javasupport/impl/PassivationStrategies.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudstate.javasupport.impl
+
+import java.time.Duration
+
+import com.typesafe.config.ConfigFactory
+import io.cloudstate.javasupport.PassivationStrategy
+
+private[impl] case class Timeout(duration: Duration) extends PassivationStrategy
+
+private[impl] object Timeout {
+
+  private val defaultPassivationTimeout: Duration = {
+    val config = ConfigFactory.load()
```

Comment:
Ideally the config is not loaded in multiple places (even though the config loading is cached), so that it's only defined once, especially if custom config can be passed into the top-level classes. Looks like `CloudStateRunner` has a constructor that accepts a config. Could we get move the defaults to one place, so they can all use the same config?

Reply:
I will take a look and see how the issue can be solved.

Reply:
> Could we get move the defaults to one place, so they can all use the same config?

@pvlugter I am not sure I understand. Could you please be more specific?